### ISSUE TITLE
[Bug] Fermenter Removal does not update when game is restarted

### ISSUE THREAD

**Issue by Yulife** [jaredlll08/ModTweaker2#38]
~~Using [this](http://paste.ee/p/LP25m) script I should be able to remove Sugar Cane from working in a Fermenter as it is the InputStack. However, even with the script, I can still ferment Sugar Cane into Biomass.~~

**Comment by Yulife** [comment]
EDIT: It seems to just be NEI Integration. NEI still shows the recipe although it is removed. @Tonius, any ideas?

**Comment by Tonius** [comment]
I'm not sure why that would be happening. As of the latest version of NEI Integration, all Forestry handlers grab the recipes straight from the machines' recipe managers every time a recipe is looked up. No caching on my side anymore.

**Comment by Yulife** [comment]
Mhmhm.. this is getting really strange.
Everytime I restart the game with the script Sugar Cane can be used in the Fermenter. However, when I do /mt reload it stops all of the sudden.

**Comment by Yulife** [comment]
Something I also noticed is that in NEI only the recipe for Sugar Cane + Water is removed. Sugar Cane + Honey / Juice still shows up in NEI.

**Comment by jaredlll08** [comment]
wat
Does it work on restart?
What happens on reload?

**Comment by Yulife** [comment]
Restart: Even with the script activated, Sugar Cane works with ALL liquids in the Fermenter.

On Reload: The Sugar Cane stops to work in the Fermenter.

**Comment by jaredlll08** [comment]
what does the script do?

**Comment by Yulife** [comment]
Remove the Fermenter recipe(s) for Sugar Cane.
Sugar Cane can be used with Water, Juice and Honey alongside of a Fermenting Agent (Fertilizer, Humus, Compost) and energy to produce Biomass.

**Comment by Yulife** [comment]
@mezz you have any ideas? This is a huge bug.

**Comment by jaredlll08** [comment]
I am unsure about what the issue is... 

ELI5 please

**Comment by Yulife** [comment]
There are 3 types of fluids in Forestry that you can use to ferment. Water, Honey and Juice. Thus, there will be 3 recipes with the InputStack, for example Water + Sapling -> Biomass, Honey + Sapling -> Biomass and Juice + Sapling -> Biomass, 
If I remove a Fermenting Recipe, [here](http://puu.sh/fxQZh/b43a6b4d30.txt) I am removing it for Sugar Cane, it still [shows](http://puu.sh/fxR3N/bbeafeffda.png) the recipes for Water + Sugar Cane -> Biomass and Juice + Sugar Cane -> Biomass in **NEI**.
In the actual **world** it does the same.
![2015-02-06_12 59 25](https://cloud.githubusercontent.com/assets/8559428/6079020/1d24c334-ae00-11e4-8b0f-c296d172e165.png)
As you can see it fermented with Water and Juice but not with Honey. I put Sugar Cane in them.
After a whole game restart, the same thing but now Water is not useable, thus in the same situation as Honey was.

**Comment by jaredlll08** [comment]
wat ELI2

**Comment by Yulife** [comment]
And now Fruit Juice doesn't work. Just check if your removal applies to all 3 recipes @jaredlll08 and stop posting stupid questions.

**Comment by Yulife** [comment]
Fixed!
